### PR TITLE
Update leaflet-draw to 0.4.12

### PR DESCRIPTION
**Pull request by tatut**
New version of leaflet-draw (0.4.12) fetched from npm registry.
I installed it locally and tested it as a dependecy.

**Extern:** The API did not change.

### CODE REVIEW

File: leaflet-draw/build.boot [hunk 1]
```diff
@@ -20,17 +20,17 @@
...
-                 #"^Leaflet.draw-(.*)/dist/leaflet.draw.js"        	"cljsjs/production/leaflet-draw.min.inc.js"
-                 #"^Leaflet.draw-(.*)/dist/leaflet.draw.css"       	"cljsjs/common/leaflet-draw.inc.css"
-                 #"^Leaflet.draw-(.*)/dist/images/spritesheet.png" 	"cljsjs/common/images/spritesheet.png"
-                 #"^Leaflet.draw-(.*)/dist/images/spritesheet-2x.png" "cljsjs/common/images/spritesheet-2x.png"
-                 }) ;; TODO Provide one-liner for images
+    (download :url      (format "https://registry.npmjs.org/leaflet-draw/-/leaflet-draw-%s.tgz" +lib-version+)
+              :decompress true
+              :compression-format "gz"
+              :archive-format "tar")
+    (sift :move {#"^.*/dist/leaflet.draw-src.js" "cljsjs/development/leaflet-draw.inc.js"
```
Comment (Deraen) [inline leaflet-draw/build.boot]: Would be good time to update this package to use convention of adding library name to the path: https://github.com/cljsjs/packages/wiki/Creating-Packages#3-create-a-directory-structure-for-the-package

Especially as this package now uses paths like `cljsjs/common/images/spritesheet.png`, which could easily clash with other package (which doesn't include lib name on path).

Comment (tatut) [reply leaflet-draw/build.boot]: Ok, I'll change paths to something like `cljsjs/leaflet_draw/common/images/*`.
Should we use the dash->underscore convention?

Comment (Deraen) [reply leaflet-draw/build.boot]: I think it is not necessary to change dashes to underscore. I think other packages just use dashes: https://github.com/cljsjs/packages/blob/master/leaflet-fullscreen/build.boot#L25

**Comment by tatut** [comment]
Changed paths, now the generated jar contents are:

```
META-INF/
META-INF/MANIFEST.MF
META-INF/maven/
META-INF/maven/cljsjs/
META-INF/maven/cljsjs/leaflet-draw/
META-INF/maven/cljsjs/leaflet-draw/pom.properties
META-INF/maven/cljsjs/leaflet-draw/pom.xml
cljsjs/
cljsjs/leaflet-draw/
cljsjs/leaflet-draw/common/
cljsjs/leaflet-draw/common/images/
cljsjs/leaflet-draw/common/images/layers-2x.png
cljsjs/leaflet-draw/common/images/layers.png
cljsjs/leaflet-draw/common/images/marker-icon-2x.png
cljsjs/leaflet-draw/common/images/marker-icon.png
cljsjs/leaflet-draw/common/images/marker-shadow.png
cljsjs/leaflet-draw/common/images/spritesheet-2x.png
cljsjs/leaflet-draw/common/images/spritesheet.png
cljsjs/leaflet-draw/common/leaflet-draw.ext.js
cljsjs/leaflet-draw/common/leaflet-draw.inc.css
cljsjs/leaflet-draw/development/
cljsjs/leaflet-draw/development/leaflet-draw.inc.js
cljsjs/leaflet-draw/production/
cljsjs/leaflet-draw/production/leaflet-draw.min.inc.js
deps.cljs
```

**Pull request merged**